convert.php

```diff
@@ -0,0 +1,72 @@
+<?php
+$languages = [
+    'Chinese' => 'zh',
+    'Chinese(Simplified)' => 'zh_CN',
+    'Czech' => 'cs_CZ',
+    'Danish' => 'da_DK',
+    'EnglishUK' => 'en_UK',
+    'EnglishUS' => 'en_US',
+    'French' => 'fr_FR',
+    'German' => 'de_DE',
+    'Greek' => 'el_GR',
+    'Italian' => 'it_IT',
+    'Japanese' => 'ja_JP',
+    'Lithuanian' => 'lt_LT',
+    'Norwegian' => 'no_NO',
+    'Polish' => 'pl_PL',
+    'Portuguese' => 'pt_PT',
```

Comment:
The current Portuguese translation was done for Brazilian Portguese so this should be pt_BR